admin/data/sql/migrations/126.do.create-pin-gen-view.sql

```diff
@@ -0,0 +1,95 @@
+CREATE VIEW [mtc_admin].vewPupilsEligibleForPinGeneration AS
+  SELECT
```

Comment:
i wonder if we will eventually want to include the check status in the query output at some point.  although i imagine that will requirement surface if necessary

Reply:
As this view is for pupils who are eligible for a pin-generation, check status fall into 3 groups:

1. no existing checks - null
2. expired checks - expired
3. for restarts the previous check could be almost anything and is largely irrelevant